src/components/KTable/KTable.vue

```diff
@@ -564,7 +580,21 @@ export default defineComponent({
       })
       data.value = res.data
       total.value = props.paginationTotalItems || res.total || res.data?.length
+
+      // get data
+      if (props.fetcher) {
+        if (props.enableClientSort && sortColumnKey.value && sortColumnOrder.value) {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
+          defaultSorter(sortColumnKey.value, '', sortColumnOrder.value, data.value)
+        }
+      } else if (props.options && props.options.data && props.options.data.length) {
+        data.value = props.options.data
+        total.value = props.options.data.length
+      }
```

Comment:
Note: @kaiarrowood We need to verify the fetching works properly in the Sandbox app